### ISSUE TITLE
The "Ribbon design" dialog should list the workbenches in the same order as the workbench toolbar

### ISSUE THREAD

**Issue by raindropsfromsky** [APEbbers/FreeCAD-Ribbon#122]
There is a major difference between the two listings. So finding the same workbench in both places is extra effort.
![image](https://github.com/user-attachments/assets/05b80153-7ca8-4601-a19e-80f301596abf)

**Desired:** 
Let the "Ribbon design" dialog list the workbenches in the same order as the workbench toolbar

**Comment by hasecilu** [comment]
In  the past I also had that issue but then I reordered some WB, and started to work as expected, not sure the cause of the problem/fix.

**Comment by APEbbers** [comment]
I haven't found the cause yet. Reordering seems to reset the cache somehow. Also it can help to set the workbench which has problems to autoload on startup.

To solve this issue, I'm going to look in the serialize function of the SearchBar addon. that addon stores the data in a file.
That technique can be a solution for this as well for the loading time of the design menu.

**Comment by raindropsfromsky** [comment]
One possible solution is to sort all WBs alphabetically in the Ribbon WB's dialog. That way, regardless of the order of the WBs in the toolbar, we can easily find any given WB.

Currently, the Ribbon list contains _three_ subsets, each of which is alphabetically sorted!
1. Assembly-Techdraw
2. Beltrami-Silk
3. Curves

(see the screenshot I posted in the first post. This listing is valid for the WBs I have selected.)

So there must be some logic behind this!
